emulator/blockchain.go

```diff
@@ -1641,3 +1664,35 @@ func (b *Blockchain) GetTransactionResultsByBlockID(blockID flowgo.Identifier) (
 	}
 	return results, nil
 }
+
+func (b *Blockchain) GetSourceFile(location common.Location) string {
```

Comment:
this isn't called in anywhere in this PR. Does this need to be hooked up to the code coverage system in some way?

Reply:
this will be separate PR from @m-Peter, to integrate it to code coverage api endpoint. ( it has to also be added to cadence testing etc )

Reply:
Yes indeed, I have already tested this on the code coverage API endpoint. See image:

![Screenshot from 2023-06-26 15-22-44](https://github.com/onflow/flow-emulator/assets/1778965/b7ac88fa-aeb0-4235-a73f-40882307a53c)

And it works as expected :tada: